tests/test_user_mgt.py

```diff
@@ -663,6 +663,23 @@ def test_update_user_valid_since(self, user_mgt_app, arg):
         request = json.loads(recorder[0].body.decode())
         assert request == {'localId': 'testuser', 'validSince': int(arg)}
 
+    @pytest.mark.parametrize('arg', [['phone'], ['google.com', 'phone']])
+    def test_update_user_delete_provider(self, user_mgt_app, arg):
+        user_mgt, recorder = _instrument_user_manager(user_mgt_app, 200, '{"localId":"testuser"}')
+        user_mgt.update_user('testuser', providers_to_delete=arg)
+        request = json.loads(recorder[0].body.decode())
+        assert set(request['deleteProvider']) == set(arg)
+
+    @pytest.mark.parametrize('arg', [['phone'], ['google.com', 'phone']])
```

Comment:
Let's also add `['google.com']` to the parameter list.

Reply:
done